packages/react/src/Blankslate/Blankslate.tsx

```diff
@@ -20,45 +25,61 @@ export type BlankslateProps = React.PropsWithChildren<{
    */
   spacious?: boolean
 
-  className?: string
-}>
+  /**
+   * Specify the variant of this component
+   */
+  variant?: 'small' | 'default' | 'large'
```

Comment:
```suggestion
  variant?: 'small' | 'medium' | 'large'
```

We would need to change this everywhere.